.github/workflows/release.yml

```diff
@@ -26,6 +27,7 @@ jobs:
     outputs:
       version: ${{ steps.version.outputs.version }}
       publish_docs: ${{ steps.check_commits.outputs.publish_docs }}
+      commit_details: ${{ steps.check_commits.outputs.commit_details }}
     steps:
       - uses: actions/checkout@v3
```

Comment:
The name of the job should be more specific and descriptive. Consider renaming it to something like "Check commit messages and descriptions for feat or break".